src/main/java/seedu/address/logic/commands/AddMedicineCommand.java

```diff
@@ -0,0 +1,16 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+public class AddMedicineCommand extends Command {
+    public static final String COMMAND_WORD = "addMedicine";
```

Comment:
Long command name, can add shorthand or alias (e.g. "aM" instead of "addMedicine") to reduce typing in the GUI